test/gtest/ucp/test_ucp_tag_match.cc

```diff
@@ -16,8 +16,10 @@ class test_ucp_tag_match : public test_ucp_tag {
 public:
     virtual void init()
     {
-        m_env.push_back(new ucs::scoped_setenv("UCX_RC_TM_ENABLE", "y"));
-        modify_config("TM_THRESH",  "1");
+        if (!RUNNING_ON_VALGRIND) {
+            m_env.push_back(new ucs::scoped_setenv("UCX_RC_TM_ENABLE", "y"));
```

Comment:
isn't there a better way to still test TM with valgrind?
for example reduce the MAX_BCOPY value or optimize something else?
In theory, TM offload should be faster with valgrind, why is it slower?

Reply:
It is slower due to many unexpected messages. 
Yes, reducing TM_MAX_BCOPY to 8192 should to the trick. Will do that instead

Reply:
still, i would assume HW TM unexpected will take as much time as SW TM, no?

Reply:
We have much bigger segment in case of HW TM now. So I thought that the following takes more time (for every HW TM capable transport): 
VALGRIND_MAKE_MEM_DEFINED when taking bcopy desc
VALGRIND_MAKE_MEM_NOACCESS when posting RX WQEs